anitya/lib/backends/gitea.py

```diff
@@ -27,7 +27,7 @@ class GiteaBackend(BaseBackend):
     with gitea api.
     """
 
-    name = "Gitea"
+    name = "Gitea/Forgejo"
```

Comment:
This is not a good idea, as it will invalidate the projects that are already have Gitea set as backend and they will not recognize the backend.

Reply:
I'll look into doing something like the following instead:

```html
<select id="backend" name="backend" required="" tabindex="3">
...
  <option value="Gitea">Gitea/Forgejo</option>
...
</select>

Reply:
Doing that in HTML should be OK